src/views/catalog/CreateFromInstanceTypes/CreateFromInstanceType.tsx

```diff
@@ -3,27 +3,59 @@ import { Trans } from 'react-i18next';
 
 import SelectInstanceTypeSection from '@catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/SelectInstanceTypeSection';
 import VMDetailsSection from '@catalog/CreateFromInstanceTypes/components/VMDetailsSection/VMDetailsSection';
+import EnableInstanceTypeTechPreviewModal from '@catalog/EnableInstanceTypeTechPreviewModal/EnableInstanceTypeTechPreviewModal';
 import HelpTextIcon from '@kubevirt-utils/components/HelpTextIcon/HelpTextIcon';
+import Loading from '@kubevirt-utils/components/Loading/Loading';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
-import { Card, Divider, Grid, GridItem, List, PopoverPosition } from '@patternfly/react-core';
+import {
+  Bullseye,
+  Card,
+  Divider,
+  Grid,
+  GridItem,
+  List,
+  PopoverPosition,
+} from '@patternfly/react-core';
 
 import AddBootableVolumeButton from './components/AddBootableVolumeButton/AddBootableVolumeButton';
 import BootableVolumeList from './components/BootableVolumeList/BootableVolumeList';
 import CreateVMFooter from './components/CreateVMFooter/CreateVMFooter';
 import SectionListItem from './components/SectionListItem/SectionListItem';
+import useEnableInstanceTypeModal from './hooks/useEnableInstanceTypeModal';
 import { useInstanceTypeVMStore } from './state/useInstanceTypeVMStore';
 import { INSTANCE_TYPES_SECTIONS } from './utils/constants';
 
 import './CreateFromInstanceType.scss';
 
-const CreateFromInstanceType: FC = () => {
+type CreateFromInstanceTypeProps = {
+  isInstanceTypeTab: boolean;
+  navigateToCatalog: () => void;
+};
+
+const CreateFromInstanceType: FC<CreateFromInstanceTypeProps> = ({
+  isInstanceTypeTab,
+  navigateToCatalog,
+}) => {
   const { t } = useKubevirtTranslation();
   const sectionState = useState<INSTANCE_TYPES_SECTIONS>(INSTANCE_TYPES_SECTIONS.SELECT_VOLUME);
 
-  const { resetInstanceTypeVMState } = useInstanceTypeVMStore();
+  const { bootableVolumesData, instanceTypesAndPreferencesData, resetInstanceTypeVMState } =
+    useInstanceTypeVMStore();
+
+  const { loading, ...enableITModalProps } = useEnableInstanceTypeModal(
+    isInstanceTypeTab,
+    navigateToCatalog,
+  );
 
   useEffect(() => resetInstanceTypeVMState(), [resetInstanceTypeVMState]);
 
+  if (loading || !bootableVolumesData?.loaded || !instanceTypesAndPreferencesData?.loaded)
+    return (
+      <Bullseye>
```

Comment:
maybe center this also in the height? like height: 100% or something?

Reply:
height 100% is not good as the PF Tab component has a class that I need to override which is causing the pages to have bigger height than wanted, I went for 50vh to get a result that looks fine at different screen heights